addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,351 @@
+        <item-type>Contact</item-type>
+        <label>Contact Channel</label>
+        <description></description>
+    </channel-type>
+
+    <channel-type id="dimminglevel-channel">
+        <item-type>Dimmer</item-type>
+        <label>Dimming Level Channel</label>
+        <description></description>
+        <state min="0" max="100" step="1" readOnly="false"></state>
+    </channel-type>
+
+    <channel-type id="mood-channel">
+        <item-type>Number</item-type>
+        <label>Mood Channel</label>
+        <description></description>
+         <state min="1" max="5" step="1" readOnly="false"></state>
+    </channel-type>
+
+    <channel-type id="status-channel">
+        <item-type>String</item-type>
+        <label>Status Channel</label>
+        <description></description>
+ 		<state readOnly="false"></state>
+    </channel-type>
+
+    <channel-type id="setpoint-channel">
+        <item-type>Number</item-type>
+        <label>Set-point Channel</label>
+        <description>Requested temperature</description>
+         <state min="0" max="255" step="1" pattern="%d °C" readOnly="false"></state>
+    </channel-type>
+
+    <channel-type id="motion-channel">
+        <item-type>Switch</item-type>
+        <label>Motion Channel</label>
+        <description>Motion detection sensor state</description>
+        <state readOnly="false"></state>
+    </channel-type>
+
+    <channel-type id="rainrate-channel">
+        <item-type>Number</item-type>
+        <label>Rain Rate Channel</label>
+        <description>Rain fall rate in millimeters per hour</description>
+        <state pattern="%d mm/h" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="raintotal-channel">
+        <item-type>Number</item-type>
+        <label>Rain Total Channel</label>
+        <description>Total rain in millimeters</description>
+        <state pattern="%d mm" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="shutter-channel">
+        <item-type>Rollershutter</item-type>
+        <label>Shutter Channel</label>
+        <description></description>
+    </channel-type>
+
+    <channel-type id="instantpower-channel">
+        <item-type>Number</item-type>
+        <label>Instant Power Channel</label>
+        <description>Instant power consumption in Watts</description>
+        <state pattern="%d W" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="totalusage-channel">
+        <item-type>Number</item-type>
+        <label>Total Usage Channel</label>
+        <description>Used energy in Watt hours</description>
+        <state pattern="%d Wh" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="instantamp-channel">
+        <item-type>Number</item-type>
+        <label>Instant Amp Channel</label>
+        <description>Instant current in Amperes</description>
+        <state pattern="%d A" readOnly="true"></state>
+    </channel-type>
+
+    <channel-type id="totalamphours-channel">
```

Comment:
You should go over all channel types and check where it might make sense to mark them as "advanced". This one here might e.g. be a good candidate.